Source/Core/UICommon/AutoUpdate.h

```diff
@@ -54,9 +54,9 @@ class AutoUpdateChecker
     NO_RESTART_AFTER_UPDATE = 0,
     RESTART_AFTER_UPDATE,
   };
-  bool TriggerUpdate(const NewVersionInformation& info, RestartMode restart_mode) const;
+  bool TriggerUpdate(const NewVersionInformation& info, const RestartMode restart_mode) const;
```

Comment:
The const qualifier for restart_mode is kind of useless *in the declaration* and should be removed.